src/components/RepoCard.tsx

```diff
@@ -0,0 +1,64 @@
+import '@styles/RepoCard.scss'
+import { RoundedTag } from './RoundedTag'
+import { Col, Row, Typography } from 'antd'
+import { StarOutlined } from '@ant-design/icons'
+import '@styles/RepoCard.scss'
+import { GithubRepoContent } from '@types'
+
+const { Title, Text } = Typography
+
+interface RepoCardProp {
+  repo: GithubRepoContent
+}
+export const RepoCard = ({ repo }: RepoCardProp) => {
+  interface EmojiMap {
+    [key: string]: string
+  }
+  const emojiMap: EmojiMap = {
+    ':eyes:': '👀',
+    ':pencil2:': '✏️',
+  }
```

Comment:
the interface and the object shouldn't live inside of RepoCard, it is bad practice in general to have interfaces and types defined inside of a component/function block

Reply:
Moved to `replaceShortcodeEmojis.ts` under `utils` folder

Reply:
I believe it should live under the `constants` folder, could you move it to keep the consistency 🙏🏼